### PR TITLE
Suggestion for additional user informations.

### PR DESCRIPTION
**Pull request by Targunitoth**
This is just a suggestion:
To provide additional hosts sources you can refer the user to the AdAway wiki: [https://github.com/AdAway/AdAway/wiki/HostsSources](https://github.com/AdAway/AdAway/wiki/HostsSources)

### CODE REVIEW

**Comment by Targunitoth** [comment]
Here is my new suggestion. I think it is the best like this. If you have a schedule you don't have to care about the `adaway-linux.sh`. If anyone is interested in more information, he will read the `README.md`.

**Comment by stablestud** [comment]
Then how about a configuration file that the user may configure prior running `adaway-linux.sh`. I've already done this in my ad-blocking script [Adsorber](https://github.com/stablestud/adsorber), and I would like to implement it if it is demanded. :blush:

**Comment by stablestud** [comment]
Just a quickinfo on Adsorber:
It was a hard fork from `adaway-linux` with changes that would be too overkill to implement into `adway-linux`.

**Comment by sedrubal** [comment]
@Targunitoth I think the easiest way was to ask the user, if he wants to run `adaway-linux.sh` now in order to inform the user that adaway is not working as long as the script did not run.

@stablestud Adsorber looks great! Well done. I started this project long time ago, when I didn't much about bash, Linux, ... Maybe it's time to drop it and redirect to Adsorber instead... What do you think?

File: install.sh [hunk 1]
```diff
@@ -185,14 +186,19 @@ EOL
                         chmod u=rw,g=r,o=r "${SYSTEMD_DIR}/adaway-linux."*
 
                         # Enable the schedule
-                        systemctl enable adaway-linux.timer && systemctl start adaway-linux.timer && echo "[i] Systemd service succesfully initialized."
+                        systemctl --quiet enable adaway-linux.timer && systemctl start adaway-linux.timer && echo "[i] Systemd service succesfully initialized."
                         ;;
                     * )
                         echo "[i] No schedule created." 1>&2
+                        echo "[i] Please run »adaway-linux.sh« to update your /etc/hosts manually." 1>&2
```
Comment (sedrubal) [inline install.sh]: I think, this line should be removed when we run adaway-linux.sh later...

**Comment by stablestud** [comment]
@sedrubal If you sure that you want to drop `adaway-linux` then I would be happy about it, as I can focus then more on `Absorber`. Give me about two weeks so that I can run proper quality assurance and update the documentation to make it clear and understandable.

**Comment by sedrubal** [comment]
@stablestud almost one year later, I found this open pull request again. Sorry for that. I'll close this merge request and archive this project while linking to yours. I partly reviewed Adsorber and I like it. Well done.

**Pull request closed without merging**